sdk/core/Azure.Core/aotcompatibility/test-aot-compat.ps1

```diff
@@ -0,0 +1,34 @@
+param([string]$targetNetFramework)
+
+$publishOutput = dotnet publish -nodeReuse:false /p:UseSharedCompilation=false /p:ExposeExperimentalFeatures=true
```

Comment:
I think you should be explicit with the `/cl` and `/clp` options to make sure you get exactly what you want. Global defaults can alter logging otherwise. The console logging options will make sure you always have what you expect.